### DISCUSSION TITLE
Is KAMP being installed correctly?

### DISCUSSION THREAD

**Discussion by spitfire** [Guilouz/Creality-Helper-Script-Wiki#500] [Issues]
I've just updated the helper script and installed KAMP using it, and I did not notice any change.
1. Not seeing anything in Heightmap menu of Mainsail (there should be new buttons there, right?)
2. Not showing up in Update manager (in Machine tab of Mainsail)
3. Printer.cfg contains include Helper-Script/KAMP/KAMP_Settings.cfg, but Helper-Script/KAMP folder is empty

**Comment by akeslo** [comment]
you would notice by the way it prints the purge line, it does it right in front of the print instead of the left side. Sounds like you don't have it setup correctly.
